### DISCUSSION TITLE
Change table cell color based on a condition

### DISCUSSION THREAD

**Discussion by Mozix** [Budibase/budibase#12511] [Help] [answered]
Hello,

Would it be possible to change the color of a cell background or the text based on a condition without using any other extra plug-ins because the plug in super table and super cell don't seem to work properly.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Mozix,

Sorry but this isn't possible with the core table component within Budibase. If there are issues with the super table component, I'd recommend creating an issue for that custom component on its repository. 

To do this in Budibase you'd need to use a repeater and the cards component, that would be the only way to customise the UI on an looped item.

Additionally you could look into this https://docs.budibase.com/docs/table-rows-status-icon

I hope this helps.

**Reply by Mozix** [reply]
Would it be possible to have the code behind the table Component in BB?
Maybe I could hard-code my conditions in by making a custom component with the statements enabled in using the Plugins features, is there a way I can find the default table component BB has or the design/code of it or would I need to write one from scratch?
